README.md

```diff
@@ -12,12 +13,13 @@ This program is aimed at short startup time, with all generally used features.
 This is a evince or qpdfview alternative for linux users.
 
 ### Installation
-To Install this program open terminal inside gospel-pdf-viewer-master directory.  
+To Install this program open terminal inside gospel-pdf-viewer directory.  
 First compile UI and Resources file  
 `cd files`  
 `./compile_ui`  
-`./compile_rc`  
+`./compile_rc`
 `cd ..`  
+`pyrcc5 -o gospel_pdf/resources_rc.py files/resources.qrc`
```

Comment:
@dammhannes why this line is required?
./compile_rc command does the same thing

Reply:
sorry, you are right!
I was overlooking it because it wouldn't run without pyqt5-dev-tools installed.

Please ignore line 22